src/anet.c

```diff
@@ -448,6 +458,17 @@ static int _anetTcpServer(char *err, int port, char *bindaddr, int af, int backl
 
         if (af == AF_INET6 && anetV6Only(err,s) == ANET_ERR) goto error;
         if (anetSetReuseAddr(err,s) == ANET_ERR) goto error;
+        if (dev) {
```

Comment:
See comment above about `ifdef`ing this out in case there's no `SO_BINDTODEVICE`.

Reply:
fixed